aggregate_temporal_period.json

```
@@ -0,0 +1,115 @@
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "hour",
+                    "day",
+                    "week",
+                    "dekad",
+                    "month",
+                    "season",
+                    "tropical-season",
+                    "year",
+                    "decade",
+                    "decade-ad"
+                ]
+            }
+        },
+        {
+            "name": "reducer",
+            "description": "A reducer to be applied on all values along the specified dimension. A reducer is a single process such as ``mean()`` or a set of processes, which computes a single value for a list of values, see the category 'reducer' for such processes.",
+            "schema": {
+                "type": "object",
+                "subtype": "process-graph",
+                "parameters": [
+                    {
+                        "name": "data",
+                        "description": "A labeled array with elements of any type.",
+                        "schema": {
+                            "type": "array",
+                            "subtype": "labeled-array",
+                            "items": {
+                                "description": "Any data type."
+                            }
+                        }
+                    },
+                    {
+                        "name": "context",
+                        "description": "Additional data passed by the user.",
+                        "schema": {
+                            "description": "Any data type."
+                        },
+                        "optional": true,
+                        "default": null
+                    }
+                ]
+            }
+        },
+        {
+            "name": "dimension",
+            "description": "The name of the temporal dimension for aggregation. All data along the dimension is passed through the specified reducer. If the dimension is not set or set to `null`, the data cube is expected to only have one temporal dimension. Fails with a `TooManyDimensions` error if it has more dimensions. Fails with a `DimensionNotAvailable` error if the specified dimension does not exist.",
+            "schema": {
+                "type": [
+                    "string",
+                    "null"
+                ]
+            },
+            "default": null,
+            "optional": true
```

```suggestion
            "optional": true,
            "default": null
```

(same order also here)

---

Order in JSON is not relavant and I doubt we have a consistent order in the processes.

---

Changed it anyway